README.md

```diff
@@ -1,27 +1,23 @@
 
-You can download the latest build [here](https://github.com/genshinsim/gcsim/releases). You can also use the webapp: [https://gcsim.app](https://gcsim.app).
+Primary usage of gcsim is through the webapp: [https://gcsim.app](https://gcsim.app). You can download the latest build and run it as a CLI [here](https://github.com/genshinsim/gcsim/releases).
 
 ## Project status
 
-The project is still under development. While every character up until 2.8 has been implemented with proper frames and hitlag, there are still improvements that can be made, which you can find in our [issues](https://github.com/genshinsim/gcsim/issues?q=is%3Aopen+is%3Aissue) and [discussions](https://github.com/genshinsim/gcsim/discussions).
+The project is still under development. While the vast majority of characters, items, and game mechanics have been implemented, there are still improvements that can be made, which you can find in our [issues](https://github.com/genshinsim/gcsim/issues?q=is%3Aopen+is%3Aissue) and [discussions](https://github.com/genshinsim/gcsim/discussions).
 
 ## Contributing
 
-If you are looking to contribute, the following are some key areas that you can help out with. If you're interested, please don't hesistate to reach out on the sim's [discord](https://discord.gg/m7jvjdxx7q)
+If you are looking to contribute, here are some key areas that you can help out with. If you're interested, please reach out to our [Discord](https://discord.gg/m7jvjdxx7q).
 
-- Comprehensive frame counts of new character's abilities, including but not limited to:
-  - Hit mark frame (not just animation frame, which is currently in the KQM library)
-  - Cooldown start frame
-  - Energy drain frame
-  - Particle proc frame (relative to hit mark)
-- Team comp damage validation between actual game play and sim. This is done by recording videos of actual gameplay footage, and then reproducing the same team/artifact/items/targets in the simulator and comparing results frame by frame. This means both comparing the damage output as well as the reactions. The simulator should be able to reproduce actual gameplay faithfully.
-- Building action list for various common team comps and comparing/validating the result vs actual in game gameplay
-- Helping with documenting the sim in the wiki
+- Comprehensive frame counts of new character's actions, methodology detailed [here](https://docs.gcsim.app/mechanics/frames/).
+- Validation between actual game play and sim results. This is done by recording footage, and then reproducing the same team/artifact/items/targets in the sim and comparing results frame by frame. This means both the damage output as well as the reactions. The simulator should be able to reproduce it faithfully.
+- Building action lists aka "rotations" for various common team comps and submitting them to our [Config Database](https://simpact.app/) via Discord.
+- Helping with documenting the sim in the wiki.
 - Further testing of in game reactions, primarily dendro.
```

Comment:
Please fix the grammar in the remaining bullet points as well: "etc...", missing commas, missing full stops, akward wording, ...